### PR TITLE
Added PHP 7.4 to Travis as allowed failure

### PR DESCRIPTION
**Pull request by CameronHall**
Hello!

I've updated travis to test the nightly version so we can see how we perform on upcoming PHP versions (e.g. PHP 7.4)

Thanks!

### CODE REVIEW

**Comment by CameronHall** [comment]
> This package requires php >=7.2 <8.0 but your PHP version (8.0.0-dev) does not satisfy that requirement.

Please hold.

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ sudo: false
 
 language: php
 php:
-  # PHP 7.4 is not ready yet
-  # - 'master'
-  - '7.3'
-  - '7.2'
+  - nightly
```
Comment (sergeyklay) [inline .travis.yml]: It is 8.0 actually. I'm not sure that we need it here in the next half year

**Review by sergeyklay — CHANGES_REQUESTED** [review]
Also disable REPORT_COVERAGE for PHP 7.4

File: .travis.yml (hunk 1 above)
Comment (CameronHall) [reply .travis.yml]: Yeah I agree. I'll remove that one as it's dependent on Zephir anyway.

**Comment by sergeyklay** [comment]
@CameronHall AppVeyor supports skip directive in commit message like `[skip appveyor]`. 
For more see https://www.appveyor.com/docs/how-to/filtering-commits/#skip-directive-in-commit-message

**Comment by CameronHall** [comment]
> Also disable REPORT_COVERAGE for PHP 7.4

How do I do that?

**Comment by sergeyklay** [comment]
```yml
before_install:
  - |

    # ...
    # ...
    # ...

    if [ "$(php-config --vernum)" -ge "70400" ]; then
      export REPORT_COVERAGE=0
    fi
```

**Comment by CameronHall** [comment]
@sergeyklay all done

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**